### PR TITLE
CSS stylesheet & Font-awesome icons added to webpage

### PR DESCRIPTION
**Pull request by agustin-pomies**

### CODE REVIEW

**Comment by franciscoaguirre** [comment]
Fix conflicts.

File: app/assets/javascripts/scriptOB.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
+document.addEventListener('DOMContentLoaded', function() {
+  var checkList = document.getElementsByClassName("check");
+  var myModal = document.getElementsByClassName("modal-background")[0];
+  for(var i = 0; i < checkList.length; i++) {
```
Comment (franciscoaguirre) [inline app/assets/javascripts/scriptOB.js]: Space after `for`.

File: app/assets/javascripts/scriptOB.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
+document.addEventListener('DOMContentLoaded', function() {
+  var checkList = document.getElementsByClassName("check");
+  var myModal = document.getElementsByClassName("modal-background")[0];
```
Comment (franciscoaguirre) [inline app/assets/javascripts/scriptOB.js]: You could use `querySelector` to not have to do the `[0]`.

File: app/assets/javascripts/scriptOB.js [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+function sleepModal() {
+  var myModal = document.getElementsByClassName("modal-background")[0];
+  myModal.style.display = "none";
+}
+
+function sortList() {
+  var list = document.getElementsByTagName("li");
+  var checkList = document.getElementsByClassName("check");
+  var j;
+  for(var i = 0; i < checkList.length; i++) {
```
Comment (rtss00) [inline app/assets/javascripts/scriptOB.js]: Use spaces after iterator keywords

File: app/assets/javascripts/scriptOB.js [hunk 4]
```diff
@@ -0,0 +1,47 @@
+document.addEventListener('DOMContentLoaded', function() {
```
Comment (guillermoap) [inline app/assets/javascripts/scriptOB.js]: why is the filename `script0B`? use a specific name that reflects it's use

File: app/assets/javascripts/scriptOB.js [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+  }
+}
+
+function sleepModal() {
+  var myModal = document.getElementsByClassName("modal-background")[0];
+  myModal.style.display = "none";
+}
+
+function sortList() {
+  var list = document.getElementsByTagName("li");
```
Comment (franciscoaguirre) [inline app/assets/javascripts/scriptOB.js]: If you have more lists in the future maybe it's best to identify this one with a class or id. What do you think?

File: app/assets/stylesheets/styleOB.css [hunk 6]
```diff
@@ -0,0 +1,116 @@
...
+  position: absolute;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  background-color: rgba(255, 255, 255, 0.4);
+}
+
+.modal {
+  position: absolute;
```
Comment (rtss00) [inline app/assets/stylesheets/styleOB.css]: Are you using a standard for the order of attributes in the stylesheets?

File: app/views/tasks/new.html.erb [hunk 7]
```diff
@@ -19,7 +18,7 @@
   <% end %>
 
   <p>
-    <%= f.text_field :description, placeholder: "Title" %>
+    <%= f.text_field :description, placeholder: "Title", :class => "text" %>
```
Comment (rtss00) [inline app/views/tasks/new.html.erb]: Use the current Ruby syntax for symbols and attribute tags, and use single quotes unless string interpolation is needed. `class: 'true` should be enough.

File: config/application.rb [hunk 8]
```diff
@@ -15,5 +15,7 @@ class Application < Rails::Application
     # Application configuration can go into files in config/initializers
     # -- all .rb files in that directory are automatically loaded after loading
     # the framework and any gems in your application.
+
+    config.assets.paths << Rails.root.join("app", "assets", "fonts") # added custom fonts
```
Comment (rtss00) [inline config/application.rb]: Single quotes.

File: app/assets/javascripts/scriptOB.js [hunk 9]
```diff
@@ -0,0 +1,47 @@
...
+          VisualExchange(list[i], list[j]);
+          break;
+        }
+        j++;
+      }
+    }
+  }
+}
+
+function VisualExchange(li1, li2) {
```
Comment (guillermoap) [inline app/assets/javascripts/scriptOB.js]: rename to `visualExchange`

File: app/assets/javascripts/scriptOB.js (hunk 3 above)
Comment (franciscoaguirre) [inline app/assets/javascripts/scriptOB.js]: You can use the `.sort` javascript built-in method. This link has more information on it: https://www.w3schools.com/jsref/jsref_sort.asp

File: app/assets/javascripts/scriptOB.js (hunk 9 above)
Comment (guillermoap) [reply app/assets/javascripts/scriptOB.js]: also avoid naming variables like x1, x2, x3. Pls give them meaningful names

File: app/assets/stylesheets/styleOB.css [hunk 10]
```diff
@@ -0,0 +1,116 @@
+@import "font-awesome";
+
+@font-face {
+  font-family: "Montserrat";
+  src: url(/assets/Montserrat-Medium.ttf) format("truetype");
+}
+
+.btn, input[type=submit] {
+  font-family: 'Montserrat', sans-serif;
```
Comment (franciscoaguirre) [inline app/assets/stylesheets/styleOB.css]: Order css properties alphabetically.

File: app/views/tasks/index.html.erb [hunk 11]
```diff
@@ -1,21 +1,21 @@
...
       </div>
     </li>
   <% end %>
-  <%= submit_tag 'Update' %>
 </ul>
-<% end %>
 
-<%= link_to 'Create task', new_task_path %>
-<%= link_to 'Log out', destroy_user_session_path, method: :delete %>
+<%= link_to 'Create task', new_task_path, :class => "btn primary" %>
```
Comment (franciscoaguirre) [inline app/views/tasks/index.html.erb]: Use rails new syntax and single quotes inside ruby: `class: 'btn primary'`

File: app/views/tasks/new.html.erb (hunk 7 above)
Comment (franciscoaguirre) [inline app/views/tasks/new.html.erb]: Rails new syntax

**Pull request merged**